### PR TITLE
:bar_chart: Create poverty projections dataset from World Bank

### PR DESCRIPTION
**Pull request by paarriagadap**
Discussed here https://github.com/owid/owid-issues/issues/1519

### CODE REVIEW

File: etl/steps/data/garden/wb/2024-06-26/poverty_projections.meta.yml
Comment (antea04) [inline etl/steps/data/garden/wb/2024-06-26/poverty_projections.meta.yml:8]: Is there a benefit to making all the key points separate? The first four points always show up together in the descriptions below, if possible, I would combine them in one variable.

File: etl/steps/data/meadow/wb/2024-06-26/poverty_projections.py
Comment (antea04) [inline etl/steps/data/meadow/wb/2024-06-26/poverty_projections.py:27]: Nitpicky: I would either multiply with a million here or rename this column only when you multiply

**Review by antea04 — APPROVED** [review]

**Review by antea04 — APPROVED** [review]

**Review by antea04 — APPROVED** [review]
Looks good from my side - I only left teeny tiny comments. Is there any reason you did not approve the charts in chart-diff? They look good from my end, so I went ahead and approved them, but you might want to take another look before you merge!

File: etl/steps/data/garden/wb/2024-06-26/poverty_projections.meta.yml
Comment (paarriagadap) [reply etl/steps/data/garden/wb/2024-06-26/poverty_projections.meta.yml]: No, there's no benefit. I thought any option would break the parameter, but I applied anchors and they worked. Thanks!

**Comment by paarriagadap** [comment]
Thank you @antea04! I actually approved them before, so I don't know if something was broken or not. I will report if it happens next time

**Pull request merged**
